css/style.css

```diff
@@ -0,0 +1,189 @@
+
+.body {
+  max-width: 1440px;
+  background: white;
+  margin: 0 60px 24px 42.72px;
+  font-family: "Montserrat";
+  color: #000;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 109px;
+  align-items: center;
+}
+
+.header__img {
+  display: block;
+}
+
+.bloks-1 {
+  flex-grow: .05;
+}
+
+.bloks-2 {
+  flex-grow: .35;
+}
+
+.bloks-3 {
+  flex-grow: .065;
+}
+
+.car__title {
+  text-align: center;
+  font-slyle: normal;
+  font-weight: 600;
+  font-size: 90px;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.car__subtitle {
+  text-align: center;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 15px;
+  margin-top: 0;
+}
+
+.car__tsl {
+  display: block;
+  height: 450px;
+  margin: auto;
+}
+
+.car {
+  background: url('../IMG/tesla bgr.png') no-repeat;
+  height: 100%;
+}
+
+.header__menu {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.header__menu_link {
+  color: #000000;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  text-decoration: none;
+  line-height: 18px;
+}
+
+.header__menu_link-red {
+  color: #e51937;
+  text-decoration: underline;
+}
+
+.link__img {
+  display: block;
+}
+
+.link__item {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 18px;
+}
+
+.link {
+  display: flex;
+  justify-content: space-between;
+}
+
+.welcome-page {
+  display: flex;
+  justify-content: space-evenly;
+  border: 5px solig red;
+}
+
+.welcome-page__block {
+  display: flex;
+  align-items: center;
+}
+
+.discrip {
+  margin-left: 13px;
+  display: flex;
+  flex-direction: column;
+}
+
+.welcome-page__speed-sec {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 35px;
+  line-height: 43px;
+}
+
+span {
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.welcome-page__speed-km {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.welcome-page__speed-car {
+  width: 46px;
+  height: 34.13px;
+}
+
+.welcome-page__driver-max {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 35px;
+  line-height: 43px;
+}
+
+.welcome-page__driver-go {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.welcome-page__button-image {
+  display: block;
+  width: 22px;
+  height: 22px;
+}
+
+.welcome-page__button {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.welcome-page__test-drive {
+  margin-left: 11px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.welcome-page__footer {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 13px;
+  line-height: 16px;
+  border: 2px solid #e51937;
+  width: 160px;
+  height: 40px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.welcome-page__footer-btn {
+  text-decoration: none;
+}
+
+.arrow {
+  text-align: center; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5566861)